plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/render/impl/RenderJob.java

```diff
@@ -197,14 +231,11 @@ protected void postRendering() {
         executor.getRenderer().setState(IRenderer.DONE);
     }
 
-    /**
-     * @see org.eclipse.core.runtime.jobs.Job#run(org.eclipse.core.runtime.IProgressMonitor)
-     */
     @Override
     protected IStatus run(IProgressMonitor monitor) {
 
-        if (!((RenderManager) getExecutor().getContext().getRenderManager()).isRenderingEnabled()) {
-            return Status.OK_STATUS;
+        if (!isRenderingEnabled()) {
+            return Status.CANCEL_STATUS;
```

Comment:
It the layer marked somehow as "not rendered". Is the canceling status an indicator for an "error". Can you describe implications of this switch?

Reply:
I revert this change in line 238.